src/main/scala/nl.knaw.dans.easy.solr4files/components/Solr.scala

```diff
@@ -39,26 +39,27 @@ trait Solr extends DebugEnhancedLogging {
   val maxFileSizeToExtractContentFrom: Double
   val solrClient: SolrClient
 
-  def createDoc(item: FileItem): Try[FileFeedback] = {
-    item.bag.fileUrl(item.path).flatMap { fileUrl =>
-      val solrDocId = s"${ item.bag.bagId }/${ item.path }"
-      val solrFields = (item.bag.solrLiterals ++ item.ddm.solrLiterals ++ item.solrLiterals)
-        .filter { case (_, v) => v.nonEmpty }
-      if (logger.underlying.isDebugEnabled) logger.debug(solrFields
-        .map { case (key, value) => s"$key = $value" }
-        .mkString("\n\t")
-      )
+  def createDoc(item: FileItem, ddm: DDM): Try[FileFeedback] = {
+    item.bag.fileUrl(item.authInfoItem.path.toString).flatMap { fileUrl =>
+      val solrDocId = item.authInfoItem.itemId
+      val solrLiterals = (item.solrLiterals ++ ddm.solrLiterals).filter { case (_, v) => v.nonEmpty }
```

Comment:
do you really mean `nonEmpty` here, or `nonBlank`? (we have this 'kind of' in the new version of dans-scala-lib)